mne/time_frequency/tests/test_csd.py

```diff
@@ -161,3 +159,188 @@ def test_compute_epochs_csd_on_artificial_data():
                     delta = 0.004
                 assert_true(abs(signal_power_per_sample -
                                 mt_power_per_sample) < delta)
+
+
+def _stack(arrays, axis=0):
+    """ Local implementation of `numpy.stack` because some tests run with
+        versions of numpy before 1.10. Once `numpy` 1.9 is no longer supported
+        in mne-python we can just use `np.stack`.
+    """
+
+    import numpy.core.numeric as _nx
+    # This code is copied from the numpy source code for now.
+    arrays = [_nx.asanyarray(arr) for arr in arrays]
+    if not arrays:
+        raise ValueError('need at least one array to stack')
+
+    shapes = set(arr.shape for arr in arrays)
+    if len(shapes) != 1:
+        raise ValueError('all input arrays must have the same shape')
+
+    result_ndim = arrays[0].ndim + 1
+    if not -result_ndim <= axis < result_ndim:
+        msg = 'axis {0} out of bounds [-{1}, {1})'.format(axis, result_ndim)
+        raise IndexError(msg)
+    if axis < 0:
+        axis += result_ndim
+
+    sl = (slice(None),) * axis + (_nx.newaxis,)
+    expanded_arrays = [arr[sl] for arr in arrays]
+    return _nx.concatenate(expanded_arrays, axis=axis)
+
+
+def test_compute_csd():
+    """Test computing cross-spectral density from ndarray
+    """
+
+    epochs, epochs_sin = _get_data()
+
+    tmin = 0.04
+    tmax = 0.15
+    tstart = np.where(epochs.times >= tmin)[0][0]
+    tend = np.where(epochs.times <= tmax)[0][-1] + 1
+    tslice = slice(tstart, tend, None)
+    picks_meeg = mne.pick_types(epochs[0].info, meg=True, eeg=True, eog=False,
+                                ref_meg=False, exclude='bads')
+
+    epochs_data = [e[picks_meeg][:, tslice] for e in epochs]
+    X = _stack(epochs_data, axis=0)
```

Comment:
how about just using np.concatenate(epochs_data, axis=0) and avoid the stack in fixes?